functional/OpenJcePlusTests/build.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0"?>
+
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+
+<project name="OpenJcePlus tests" default="build" basedir=".">
+	<taskdef resource="net/sf/antcontrib/antlib.xml" />
+	<description>
+		OpenJCEPlus Tests
+	</description>
+
+	<!-- set global properties for this build -->
+	<property name="DEST" value="${BUILD_ROOT}/functional/OpenJcePlusTests" />
+
+	<!--Properties for this particular build-->
+	<property name="src" location="./OpenJCEPlus/src/" />
```

Comment:
I believe that we should be compiling and putting into a jar file just the test directories. I am not quite sure what would happen if we compile both the openjceplus provider itself and the test code like this. When the tests run do they test the OpenJCEPlus module included in the build itself or would they test what you just compiled? Id hope they test what is in the Semeru build itself. 

To be safe should we be compiling just the files in the src/test directory here?

Or perhaps line 52 handles that and removes everything but the test directory?

Reply:
Yes, we only compile tests and put them openjceplus-tests.jar into https://github.com/adoptium/aqa-tests/blob/575f6fa2914cc0d8428c53a6559565fe8c630440/functional/OpenJcePlusTests/build.xml#L81

line 52 removed everything but the test directory.